test/gurka/gurka.h

```diff
@@ -161,15 +161,15 @@ valhalla::Api do_action(const valhalla::Options::Action& action,
                         const map& map,
                         const std::string& request_json,
                         std::shared_ptr<valhalla::baldr::GraphReader> reader = {},
-                        std::string* json = nullptr);
+                        std::string& json = *std::unique_ptr<std::string>(new std::string()));
```

Comment:
couldn't make it work with a string pointer, segfaulted for me when passing a string pointer..

Reply:
made it work..